src/main/java/de/jcm/discordgamesdk/activity/ActivityType.java

```diff
@@ -16,11 +16,6 @@
 	PLAYING,
 	STREAMING,
 	LISTENING,
-	WATCHING,
```

Comment:
Please add a spacer enum to make up for skipped number.
See my comment to the pull request for more details.

Reply:
I added a new value to the enum as a spacer.